src/main/java/org/jabref/gui/texparser/ParseTexResultViewModel.java

```diff
@@ -0,0 +1,31 @@
+package org.jabref.gui.texparser;
+
+import javafx.beans.property.ReadOnlyListWrapper;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+import org.jabref.gui.AbstractViewModel;
+import org.jabref.model.texparser.Citation;
+import org.jabref.model.texparser.TexParserResult;
+
+public class ParseTexResultViewModel extends AbstractViewModel {
+
+    private final ObservableList<ReferenceWrapper> referenceList;
+    private final ObservableList<Citation> citationList;
+
+    public ParseTexResultViewModel(TexParserResult texParserResult) {
+        this.referenceList = FXCollections.observableArrayList();
+        this.citationList = FXCollections.observableArrayList();
+
+        texParserResult.getCitationsKeySet().forEach(key -> referenceList.add(
+                new ReferenceWrapper(key, texParserResult.getCitationsByKey(key))));
+    }
+
+    public ObservableList<ReferenceWrapper> getReferenceList() {
+        return new ReadOnlyListWrapper<>(referenceList);
+    }
+
+    public ObservableList<Citation> getCitationList() {
```

Comment:
Please rename this so that it reflects that the citations correspond to the active reference (e.g. `getCitationsForCurrentReference` or something similar).